### PR TITLE
Fixing some typo and add break style for better experience reading

### PR DESCRIPTION
**Pull request by cacing69**

### CODE REVIEW

File: A-concurrency-pipeline.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Analoginya seperti ini: bayangkan sebuah flow proses untuk auto backup database
 1. Kita perlu data *list* dari semua database yang harus di-backup, beserta alamat akses dan kredensial-nya.
 2. Kita jalankan proses backup, bisa secara sekuensial (setelah `db1` selesai, lanjut `db2`, lanjut `db3`, dst), atau secara paralel (proses backup `db1`, `db2`, `db3`, dan lainnya dijalankan secara bersamaan).
 3. Di masing-masing proses backup database sendiri ada beberapa proses yang dijalankan:
-   A. Lakukan operasi *dump* terhadap database, outputnya berupa banyak file disimpan ke sebuah folder.
-   B. File-file hasil dump kemudian di-*archive* ke bentuk `.zip` atau `.tar.gz` (misalnya).
-   C. File archive di-kirim ke server backup, sebagai contoh AWS S3.
+    * A. Lakukan operasi *dump* terhadap database, outputnya berupa banyak file disimpan ke sebuah folder.
```
Comment (novalagung) [inline A-concurrency-pipeline.md:16]: for this section, can you please adjust it like below? simply add a new line between the `3.` and `A.`.

3. Di masing-masing proses backup database sendiri ada beberapa proses yang dijalankan:

    A. Lakukan operasi dump terhadap database, outputnya berupa banyak file disimpan ke sebuah folder.
    B. File-file hasil dump kemudian di-archive ke bentuk .zip atau .tar.gz (misalnya).
    C. File archive di-kirim ke server backup, sebagai contoh AWS S3.

File: A-concurrency-pipeline.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Semoga cukup jelas ya. Gpp kalau bingung, nanti kita sambil praktek juga jadi bi
 
 Ok, penjabaran teori sepanjang sungai `nil` tidak akan banyak membawa penjelasan yang real kalau tidak diiringi dengan praktek. So, mari kita mulai praktek.
 
-Penulis sarankan untuk benar-benar memahami setiap bagian praktek ini, karena topik ini merupakan pembahasan yang cukup berat untuk pemulai, tapi masih dalam klasifikasi fundamental kalau di Go programming. Bingung tidak apa, nanti bisa di-ulang-ulang, yang penting tidak sekadar *copy-paste*.
+Penulis sarankan untuk benar-benar memahami setiap bagian praktek ini, karena topik ini merupakan pembahasan yang cukup berat untuk pemula, tapi masih dalam klasifikasi fundamental kalau di Go programming. Bingung tidak apa, nanti bisa di-ulang-ulang, yang penting tidak sekedar *copy-paste*.
```
Comment (novalagung) [inline A-concurrency-pipeline.md:55]: The word `sekadar` is actually a valid Indonesian word :) Please revert this one. https://kbbi.web.id/sekedar%20atau%20sekadar

Comment (cacing69) [reply A-concurrency-pipeline.md:55]: aah i'm sorry for that, i will revert asap

**Comment by novalagung** [comment]
Great! LGTM

**Comment by novalagung** [comment]
@cacing69 thank you for the contribution

**Pull request merged**